File: csharp-netstandard/src/Cloud.Governance.Client/Model/ConflictSolutionType.cs


using System;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.IO;
using System.Runtime.Serialization;
using System.Text;
using System.Text.RegularExpressions;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using Newtonsoft.Json.Linq;
using System.ComponentModel.DataAnnotations;
using OpenAPIDateConverter = Cloud.Governance.Client.Client.OpenAPIDateConverter;

namespace Cloud.Governance.Client.Model
{
    
    public enum ConflictSolutionType
    {
        Skip = 0,

        NotOverWrite = 1,

        Append = 2,

        OverWrite = 3,

        Replace = 4,

        Merge = 5,

        OverWritebyLastModifiedTime = 6,

        None = 7

    }

}

File: typescript/src/models/ConflictSolutionType.ts
/* tslint:disable */
/* eslint-disable */
/*
 * Cloud Governance Api
 *
 * Contact: support@avepoint.com
 */

/**
 * 
 * @export
 * @enum {string}
 */
export enum ConflictSolutionType {
    Skip = 0,
    NotOverWrite = 1,
    Append = 2,
    OverWrite = 3,
    Replace = 4,
    Merge = 5,
    OverWritebyLastModifiedTime = 6,
    None = 7
}

export function ConflictSolutionTypeFromJSON(json: any): ConflictSolutionType {
    return ConflictSolutionTypeFromJSONTyped(json, false);
}

export function ConflictSolutionTypeFromJSONTyped(json: any, ignoreDiscriminator: boolean): ConflictSolutionType {
    return json as ConflictSolutionType;
}

export function ConflictSolutionTypeToJSON(value?: ConflictSolutionType | null): any {
    return value as any;
}

